### PR TITLE
Add in README about installing on windows.

### PR DESCRIPTION
**Pull request by jonnung**
Installing on Windows with the same procedure as phpunit.phar installation was executed.
So I added the progress in the README file.

### CODE REVIEW

**Comment by keradus** [comment]
How about leaving in readme file only info how to install globally via composer (for both Unix/Windows) and just one sentence that one can also download phar version ?
Then no need for so much text about installing on Windows (btw, I use Windows and install globally via composer) or homebrew (btw, extra outdated 1.2 version... we should not encourage to use it)

File: README.rst [hunk 1]
```diff
@@ -74,6 +74,30 @@ already have it.
 
     $ brew install homebrew/php/php-cs-fixer
 
+Globally (Windows)
+~~~~~~~~~~~~~~~~~~
+Globally installing the PHAR involves the same procedure as manually `installing PHPUnit on Windows`_:
+
+1. Create a directory for PHP binaries; e.g., ``C:\bin``
```
Comment (GrahamCampbell) [inline README.rst]: Why would you do that? I don't do that and I'm on windows.

**Comment by gharlan** [comment]
> How about leaving in readme file only info how to install globally via composer (for both Unix/Windows) and just one sentence that one can also download phar version ?

:+1:

File: README.rst (hunk 1 above)
Comment (GrahamCampbell) [inline README.rst]: It's generally a bad idea to put things in a folder on the root of the drive.

**Comment by jonnung** [comment]
It seems like good idea to use Composer. :smiley: :+1:
I know how to use the Composer but i'm not always use the Composer. So I thought solving hard problem.

**Pull request closed without merging**